FV3/atmos_cubed_sphere/tools/fv_diagnostics.F90

```diff
@@ -462,6 +466,8 @@ subroutine fv_diag_init(Atm, axes, Time, npx, npy, npz, p_ref)
        if (id_lont > 0) used = send_data(id_lont, rad2deg*Atm(n)%gridstruct%agrid(isc:iec,jsc:jec,1), Time)
        if (id_latt > 0) used = send_data(id_latt, rad2deg*Atm(n)%gridstruct%agrid(isc:iec,jsc:jec,2), Time)
        if (id_area > 0) used = send_data(id_area, Atm(n)%gridstruct%area(isc:iec,jsc:jec), Time)
+       if (id_dx > 0) used = send_data(id_dx, Atm(n)%gridstruct%dx(isc:iec,jsc:jec+1), Time)
+       if (id_dy > 0) used = send_data(id_dy, Atm(n)%gridstruct%dy(isc:iec+1,jsc:jec), Time)
```

Comment:
There's no harm in sending more diagnostics, but I am curious why? And does it relate to coarsening restart files?

Reply:
Having these diagnostics is useful with respect to coarse-graining for two reasons:
1. It allows us to verify that these coarse-graining routines work (e.g. I use these in [the notebook I linked to above](https://github.com/VulcanClimateModeling/explore/blob/master/spencerc/2020-04-20/2020-04-20-dycore-restart-coarse-graining.ipynb)).  
2. We use these more generally when further coarse-graining data offline.  Eventually I may add coarse-versions of these diagnostics too, but I will save that for a future PR, where I will add coarse versions of all the grid metrics, e.g. area, dx, dy, grid_lont, grid_latt, grid_lon, and grid_lat.